os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/i2s_api.c

```diff
@@ -507,7 +510,7 @@ void i2s_deinit(i2s_t *obj)
 	case PLL_CLOCK_98P304M:
 		PLL_I2S_98P304M(DISABLE);
 		break;
-	}
+	}*/
 
 	if (obj->i2s_idx == I2S2) {
```

Comment:
In this deinit function it only does SPORT clock disable.
Do we need GDMA_ChnlFree() and AUDIO_SP_Deinit()?
Both are called in i2s_disable() when PM try to suspend I2S activity.